### DISCUSSION TITLE
Only pull down my package when initializing ISV Debugger session

### DISCUSSION THREAD

**Discussion by Nickz22** [forcedotcom/salesforcedx-vscode#5793] [Ideas]
**Is your feature request related to a problem? Please describe.**
I'm attempting to use the ISV debugger for the first time. Project initialization goes well until I get to Step 7, at which point installed packages in the subscriber org are converted to sfdx format. A handful of packages installed on the subscriber org are not converting cleanly to sfdx, which halts the project initialization process. 

```
Processing package: TaskRay
Starting SFDX: ISV Debugger Setup, Step 7 of 7: Converting package: TaskRay

12:25:41.879 sfdx force:mdapi:convert --rootdir .sfdx/tools/isvdebuggermdapitmp/packages/TaskRay --outputdir .sfdx/tools/installed-packages/TaskRay
(node:87363) Warning: The .forceignore file doesn't adhere to .gitignore format which will be the default behavior starting in Spring '21 release. More information on .gitignore format here: https://git-scm.com/docs/gitignore. Fix the following lines in your .forceignore and add '# .forceignore v2' to your .forceignore file to switch to the new behavior.
(node:87363) Warning: 	package.xml
ERROR running force:mdapi:convert:  An error was encountered processing path: /Users/nicholaszozaya/Salesforce/package_guidedselling/src/scripts/apex/debugger/RingDNA Full/.sfdx/tools/isvdebuggermdapitmp/packages/TaskRay/lwc/trLtngOutDataFetcher/lwc/trLDSResetter/trLDSResetter.js
12:25:44.752 sfdx force:mdapi:convert --rootdir .sfdx/tools/isvdebuggermdapitmp/packages/TaskRay --outputdir .sfdx/tools/installed-packages/TaskRay
 ended with exit code 1
```

**Describe the solution you'd like**
I'd like to just download the package I own and am logged into via LMA, or the ability to ignore certain packages for Steps 6 & 7 of the project initialization process. 

**Describe alternatives you've considered**
I don't know what to do other than uninstall the problem packages, but that is far less than ideal given that most packages have dependencies which have to be undone prior to uninstall, and, since these are client sandboxes we're working in, uninstalling is not a great option. 

**Additional context**
n/a

**Comment by jonathanwiesel** [comment]
For us this would be extremely useful, there are organizations that have too many packages that the process is trying to retrieve and it takes an eternity, it could even fail if retrieving too much with an error like the following:

`The metadata retrieve operation failed: LIMIT_EXCEEDED: The retrieved zip file exceeded the limit of 629145600 bytes.`

By default the ideal would be to only pull your package, or at least give the flexibility to choose the packages
